crates/meroctl/src/cli/context.rs

```diff
@@ -47,6 +54,8 @@ impl ContextCommand {
             ContextSubCommands::List(list) => list.run(args).await,
             ContextSubCommands::Create(create) => create.run(args).await,
             ContextSubCommands::Join(join) => join.run(args).await,
+            ContextSubCommands::Get(get) => get.run(args).await,
+            ContextSubCommands::Delete(delete) => delete.run(args).await,
```

Comment:
It would be nice to keep things like this in alphabetical order as we build them up 🙂